OpenRA.Mods.Common/UpdateRules/Rules/20180923/AddSupportPowerBotModule.cs

```diff
@@ -0,0 +1,95 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class AddSupportPowerBotModule : UpdateRule
```

Comment:
How ugly would it be to combine these all into a single `ExtractHackyAIModules` rule?
It seems silly to have N different classes that are half duplicates of eachother.